### PR TITLE
Bug 1128806 - [Bluetooth] use NotificationHelper and remove mozL10n.get's from bluetooth app, r=tim

### PR DESCRIPTION
**Pull request by gasolin**
WIP

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=df9e9353f4a60e53892874104c32e8fd9558ff98

File: apps/bluetooth/js/modules/pair_manager.js [hunk 1]
```diff
@@ -282,21 +284,20 @@ define(function(require) {
         showPairviewCallback: this.showPairview.bind(this, pairingInfo)
       };
       // Prepare notification toast.
-      var title = _('bluetooth-pairing-request-now-title');
-      var body = pairingInfo.evt.deviceName || _('unnamed-device');
+      var body = pairingInfo.evt.deviceName || 'unnamed-device';
```
Comment (zbraniecki) [inline apps/bluetooth/js/modules/pair_manager.js]: I would recommend storing raw string differently than l10nId:

```
var body = pairingInfo.evt.deviceName ? {raw: pairingInfo.evt.deviceName} : 'unnamed-device';
```

as per: https://developer.mozilla.org/en-US/Firefox_OS/Developing_Gaia/localization_code_best_practices#Writing_APIs_that_operate_on_L10nIDs

File: apps/bluetooth/js/modules/pair_manager_v1.js [hunk 2]
```diff
@@ -93,21 +91,20 @@ define(function(require) {
         showPairviewCallback: this.showPairview.bind(this, pairingInfo)
       };
       // Prepare notification toast.
-      var title = _('bluetooth-pairing-request-now-title');
-      var body = pairingInfo.name || _('unnamed-device');
+      var body = pairingInfo.name || 'unnamed-device';
```
Comment (zbraniecki) [inline apps/bluetooth/js/modules/pair_manager_v1.js]: same here

File: apps/bluetooth/js/modules/pair_manager_v1.js [hunk 3]
```diff
@@ -191,7 +188,7 @@ define(function(require) {
       var evt = pairingInfo;
       var device = {
         address: evt.address,
-        name: evt.name || _('unnamed-device'),
+        name: evt.name || 'unnamed-device',
```
Comment (zbraniecki) [inline apps/bluetooth/js/modules/pair_manager_v1.js]: and here.

File: apps/bluetooth/js/modules/pair_manager.js (hunk 1 above)
Comment (gasolin) [inline apps/bluetooth/js/modules/pair_manager.js]: thanks its useful!

**Pull request closed without merging**